ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/base-fee-input/base-fee-input.js

```diff
@@ -72,7 +72,8 @@ const BaseFeeInput = () => {
     setMaxFeePerGas,
   } = useAdvancedGasFeePopoverContext();
 
-  const { estimatedBaseFee } = gasFeeEstimates;
+  const { estimatedBaseFee, historicalBaseFeeRange } = gasFeeEstimates;
```

Comment:
Do you think it's worth it to account for the case in which `gasFeeEstimates` is empty (say, the modal loads but for some reason we haven't fetched estimates yet) so we don't get a runtime error? I added an `if` statement for the priority fee range in the network stats. Not sure if we want to do that here too?